.github/workflows/kubesec.yml

```diff
@@ -13,7 +16,7 @@ jobs:
       contents: read
```

Comment:
Curious to know why do we need top level permission set if we have set it here in more granular steps.

Reply:
So why not `permissions: {} `at the top level and let the jobs decide what it needs?

Reply:
Avoids repetition. But it could be done like that too.

Reply:
I am fine with the current change as well. Was curious to know the impact of different configurations.
/approve